specs/~inheritance.yml

```diff
@@ -224,3 +224,92 @@ tests:
     partials:
       parent: "{{$foo}}default content{{/foo}}"
     expected: default content
+
+  - name: Standalone parent
+    desc: A parent's opening and closing tags need not be on separate lines in order to be standalone
+    data: {}
+    template: |
+      Hi,
+        {{<parent}}{{/parent}}
+    partials:
+      parent: |
+        one
+        two
+    expected: |
+      Hi,
+        one
+        two
+
+  - name: Standalone block
+    desc: A block's opening and closing tags need not be on separate lines in order to be standalone
+    data: {}
+    template: |
+      {{<parent}}{{$block}}
+      one
+      two{{/block}}
+      {{/parent}}
+    partials:
+      parent: |
+        Hi,
+          {{$block}}{{/block}}
+    expected: |
+      Hi,
+        one
+        two
+
+  - name: Block reindentation
+    desc: Block indentation is removed at the site of definition and added at the site of expansion
+    data: {}
+    template: |
+      {{<parent}}{{$block}}
+          one
+          two
+      {{/block}}{{/parent}}
+    partials:
+      parent: |
+        Hi,
+          {{$block}}
+          {{/block}}
```

Comment:
Here, I'm avoiding the double newline problem by making the block tag and the end section tag standalone by themselves, so their trailing newlines are removed from the output.